### ISSUE TITLE
Add a mobile alternate template template

### ISSUE THREAD

**Issue by andremalenfant** [kolber/stacey#78]
Add the ability to switch template for mobile and tablets.

I integrated the Mobile_Detect.php module to my copy of stacey to switch template and have conditions based on the browser. It would be a nice addition to the CMS.

**Comment by AdrienneLove** [comment]
I'm not a contributor to the project but I would encourage people to use responsive designs compared to mobile switches. Simply because you can test it in a desktop browser, you don't have to detect for obscure mobile devices and it's based on the size of the users screen. Though I'll admit it is something you have to keep in mind when designing a website compared to an afterthought.

**Comment by andremalenfant** [comment]
You are right AdrianLove that using media queries and client side techniques is preferable nowadays but there are situations where you want a completely different design or different functionalities on mobile and having some control on the server is necessary. The W3C recommends to use server-side detection when possible and then client side. The reason is to ensure the delivery of only necessary data for transport on a potentially slower, with lower bandwidth connection. I know the recommandations are from 2010 but most of it is still valid. There is no need to have the browser fetch big javascript libraries and large size images if the mobile design is to be simpler and the screen can only display small images. It will only lead to a less responsive application. (http://www.w3.org/TR/2010/REC-mwabp-20101214/).
